packages/zowe-explorer/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
+- Added a notification with an option to create a new configuration when a user does not have any Zowe client configurations. [#3148](https://github.com/zowe/zowe-explorer-vscode/pull/3148)
```

Comment:
Can we reword to something like: Users can now accept/use/follow a prompt to create a new Zowe client configuration if they don't already have any.

Also would be good to say here ^^ _when_ this notification is expected to take place. And: what is a "client configuration"? a config file or a profile?

Reply:
Good points - addressed in 7c3f0c7